docs/cri/crictl.md

```diff
@@ -17,6 +17,10 @@ A helper command has been included to install the dependencies at the right vers
 ```console
 $ make install-deps
 ```
+```make install-deps``` will install **libseccomp** package and **runc**,**cni** Component. On Linux system, you should install **build-essential** **gperf** package.
+```shell
+sudo apt install build-essential gperf -y
```

Comment:
build-essential is requirement to install make, but gperf?

Also In the beginning of the doc it assumes containerd is running so using make install-deps just to install cri which installs runc cni again doesn't seem right, I would suggest to make a new target `make install-critools` which would only install cri tools using the script https://github.com/containerd/containerd/blob/main/script/setup/install-critools